addons/point_of_sale/static/src/js/screens.js

```diff
@@ -2368,6 +2376,15 @@ var PaymentScreenWidget = ScreenWidget.extend({
         }
 
         order.initialize_validation_date();
+        if(order.get_rounding_applied()) {
+            var product  = this.pos.db.get_product_by_id(1);   //FIXME: Get correct rounding product
```

Comment:
You'll probably need to define a rounding product in the pos config, similarly to the discount product